Packages/Blockchain/Sources/Ethereum/Lido/EIP712Message.swift

```diff
@@ -0,0 +1,78 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+
+public struct EIP712Domain: Codable {
```

Comment:
Move to gemstone?

Reply:
will do it after block explorer pr merged in core